### PR TITLE
DynamoDB: Omit case nested arrays

### PR DESCRIPTION
**Pull request by sutt0n**
Omitted `documents`, `docketRecord`, `irsPractitioners`, and `privatePractitioners` props from case item

### CODE REVIEW

File: shared/src/persistence/dynamo/cases/createCase.js [hunk 1]
```diff
@@ -15,7 +16,12 @@ exports.createCase = async ({ applicationContext, caseToCreate }) => {
       Item: {
         pk: `case|${caseToCreate.caseId}`,
         sk: `case|${caseToCreate.caseId}`,
-        ...caseToCreate,
+        ...omit(caseToCreate, [
```
Comment (rachaelparris) [inline shared/src/persistence/dynamo/cases/createCase.js:19]: Is this still a valid case with all of these things removed? We validate before returning from the interactors.

Comment (sutt0n) [reply shared/src/persistence/dynamo/cases/createCase.js:19]: Yeah. We aggregate all necessary items when returning a case (e.g. `getCaseByCaseId`) and this method returns `caseToUpdate`, which also includes those validated properties

**Review by rachaelparris — APPROVED** [review]

**Review by codyseibert — APPROVED** [review]

File: shared/src/business/useCases/users/updateUserContactInformationInteractor.test.js [hunk 2]
```diff
@@ -93,7 +93,7 @@ describe('updateUserContactInformationInteractor', () => {
     };
   });
 
-  it("should throw an error when the user's contact information has not changed", async () => {
+  it.skip("should throw an error when the user's contact information has not changed", async () => {
```
Comment (codyseibert) [inline shared/src/business/useCases/users/updateUserContactInformationInteractor.test.js]: should these be skipped?

File: shared/src/persistence/dynamo/cases/getCasesByUser.js [hunk 3]
```diff
@@ -4,13 +4,21 @@ const {
...
+  userId,
+}) => {
   let cases = await getRecordsViaMapping({
     applicationContext,
     pk: `user|${userId}`,
     prefix: 'case',
   });
 
+  if (printOut) {
+    console.log('LE USER CASES', cases);
```
Comment (codyseibert) [inline shared/src/persistence/dynamo/cases/getCasesByUser.js]: remove if not needed

**Review by matthopson — APPROVED** [review]

**Pull request merged**